charts/fluid/fluid/VERSION

```diff
@@ -98,3 +98,12 @@ release-2.7.3-SNAPSHOT-a7154f1
 
 Version: https://github.com/Alluxio/alluxio/commit/a7154f19de408de1395c2c4985e0ed9809e6c3c1
 Branch: https://github.com/Alluxio/alluxio/tree/release-2.7.3
+
+
+20220510:
+
+release-2.8.1-SNAPSHOT-0433ade
+
+Version: https://github.com/Alluxio/alluxio/commit/0433aded47b69a9bc34bdf7676cb3224a1ddb2a1
+Branch: https://github.com/Alluxio/alluxio/tree/release-2.8.0
```

Comment:
Wondering why this is refer to branch 2.8.0 instead of 2.8.1?

Reply:
I just keep the same version with alluxio latest commit in release 2.8.0: https://github.com/Alluxio/alluxio/commit/0433aded47b69a9bc34bdf7676cb3224a1ddb2a1